app/javascript/App/components/NewsfeedPage/CoinListItem.js

```diff
@@ -20,11 +21,12 @@ export default (props) => {
         <div className="b f5 pl2">{coin.get('symbol')}</div>
         {coin.get('market_info') && (
           <div className="right-align">
-            <Currency>{coin.getIn(['market_info', 'price_usd'])}</Currency>
-            <PercentageChange
+            {coinPrice && <Currency>{coinPrice}</Currency>}
```

Comment:
@hanchang at this point the `coinPrice` gets passed into the `<Currency />` which runs `parseFloat`. i think this is a bad idea, because the are not traditional fiat currencies and fractions of a cent are meaningful. is there a reason we don't show the crypto price?